tools/datamart_benchmarker.py

```diff
@@ -11,28 +12,36 @@
 
 log = logging.getLogger(__name__)
 
-DATAMART_ROOT_URL = 'http://localhost:8844'
+LOCAL_DATAMART_ROOT_URL = 'http://localhost:8844'
+DEV_DATAMART_ROOT_URL = 'https://dev.hapi-humdata-org.ahconu.org'
+
 APP_IDENTIFIER = 'SERYSU5URVJOQUxfZG9jc191aV9paDppYW4uaG9wa2luc29uQGh1bWRhdGEub3Jn'
```

Comment:
you should probably avoid hardcoding your email here.

Reply:
Good spot - I don't actually use APP_IDENTIFIER, and we don't use the email address provided so I put in datamart@humdata.org